coordinator/src/main/java/org/apache/uniffle/coordinator/web/servlet/DecommissionServlet.java

```diff
@@ -42,9 +42,7 @@ protected Response<Object> handlePost(HttpServletRequest req, HttpServletRespons
       return Response.fail("Parameter[serverIds] should not be null!");
     }
     ClusterManager clusterManager = coordinator.getClusterManager();
-    params.getServerIds().forEach((serverId) -> {
-      clusterManager.decommission(serverId);
-    });
+    params.getServerIds().forEach(clusterManager::decommission);
```

Comment:
Emm... This is not related with this issue. Right?

Reply:
Thank you very much for your help in reviewing the code! This change does not belong to this issue, I will fix it.